Lib/test/test_spwd.py

```diff
@@ -68,7 +69,8 @@ def test_getspnam_exception(self):
         except KeyError as exc:
             self.skipTest("spwd entry %r doesn't exist: %s" % (name, exc))
         else:
-            self.assertEqual(str(cm.exception), '[Errno 13] Permission denied')
+            self.assertEqual(str(cm.exception),
```

Comment:
I think you can simply delete this assert. What it covers is already handled by catching `PermissionError` above.